### PR TITLE
fix: fixed violation tiles to display correct data when one of the tiles has 0 values.

### PR DESCRIPTION
**Pull request by sidharthjain-zemoso**
# Description
1. Fixed violation tiles to display correct data when asset group is switched.
2. Fixed breadcrumb to show up when navigated to asset list from dashboard/policy compliance.
3. Fixed asset trend graph to show first tick value.

Fixes # (issue)
Dashboard violation tiles.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] The commit message/PR follows our guidelines

### CODE REVIEW

**Review by ranadheer-b — APPROVED** [review]
LGTM

File: webapp/src/app/pacman-features/modules/compliance/card/card.component.ts [hunk 1]
```diff
@@ -63,6 +63,8 @@ export class CardComponent implements OnInit {
   
 
   handleGraphIntervalSelection = (e) => {
+    this.fromDate = new Date(2022, 1, 1);
+    this.toDate = new Date(2200, 12, 31);
```
Comment (dheerajkholia-paladin) [inline webapp/src/app/pacman-features/modules/compliance/card/card.component.ts:67]: Shouldn't we remove this hard coded dates?

Comment (sidharthjain-zemoso) [reply webapp/src/app/pacman-features/modules/compliance/card/card.component.ts:67]: Let's go with this for now? I have noted it down.

Comment (santhosh-challa) [reply webapp/src/app/pacman-features/modules/compliance/card/card.component.ts:67]: lets revisit this later.

**Review by santhosh-challa — APPROVED** [review]

**Pull request merged**